roles/pulp_django_user/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Interacting with Django requires us to use cookies for CSRF and login.
+# It also requires the use of a Referer header.
+
+- name: Get a CSRF token
+  uri:
+    url: "{{ pulp_login_url }}"
+    method: GET
+    headers:
+      Referer: "{{ pulp_login_url }}"
+    validate_certs: "{{ pulp_validate_certs | bool }}"
+  register: result_csrf
+
+- name: Login
```

Comment:
Won't this print the password in output?

Reply:
Only in verbose mode